### PR TITLE
Add sample supervisor config

### PR DESCRIPTION
**Pull request by mvdbeek**
for galaxy served by uwsgi, with separate handlers.

There was some confusion as to how to start uwsgi correctly on IRC yesterday, I hope this will help people struggling with this. Also, supervisor is a common way to start galaxy, so I guess it would be a good idea to have a sample. Ping @dannon

### CODE REVIEW

**Comment by mvdbeek** [comment]
The toolshed failure should be unrelated.

File: contrib/galaxy_supervisor.conf [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+# If this is not the case, adjust the path to uwsgi accordingly.
+# If uwsgi was not installed through pip, include "--plugin python" in the uwsgi command.
+# This configuration has been tested with galaxy release_16.01 and uWSGI==2.0.12.
+# This assumes galaxy is installed in /home/galaxy/galaxy, so change occurences of /home/galaxy/galaxy accordingly.
+# If you want to run galaxy under a different username, change "user = galaxy" to "user = <your user>".
+# You will probably want to proxy uwsgi (you can't connect with a browser to uwsgi port 4001) with nginx or apache, 
+# see https://wiki.galaxyproject.org/Admin/Config/Performance/Scaling#uWSGI-1
+# or https://wiki.galaxyproject.org/Admin/Config/Performance/Scaling#uWSGI-2 . 
+
+[program:uwsgi]
```
Comment (dannon) [inline contrib/galaxy_supervisor.conf]: s/uwsgi/galaxy_web/ ?  (or rename below in the groups definition)

Comment (mvdbeek) [inline contrib/galaxy_supervisor.conf]: right, thanks!

**Comment by dannon** [comment]
+1, thanks!  this should be useful for folks.

**Comment by martenson** [comment]
GitHub shows me that the last commit (Change program name to galaxy_web) has been added after the merge. :/ (but it is correctly in the codebase)

**Comment by martenson** [comment]
On the side note despite this being milestoned to 16.07 it will be in 16.04 becase it was merged into dev pre-freeze. Please don't merge new features in dev until we freeze&branch the release.

**Comment by dannon** [comment]
This is not a new feature, it's documentation.

**Comment by martenson** [comment]
@dannon correct, sorry for the confusion, just a warning because the logic behind might not be clear

**Comment by dannon** [comment]
Sure.  I dunno what's up with github displaying the merge before the galaxy_web name fix, I definitely merged it after (I suggested the tweak after all) and it's in the codebase.

**Pull request merged**